### PR TITLE
reviews component to antd timeline transfare and bug fix

### PR DESCRIPTION
**Pull request by costashush**
image white space elimination
reviews to antd timeline transfare

### CODE REVIEW

File: src/components/Reviews/Review.js [hunk 1]
```diff
@@ -2,44 +2,31 @@ import React from 'react';
...
-`;
-
 const ReviewWrapper = styled.div`
   margin-top: 2em;
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-  padding: 40px;
+  padding: 20px;
   border-radius: 10px;
-  margin-left: 100px;
+  margin-left: 10px;
```
Comment (andreykishtov) [inline src/components/Reviews/Review.js]: if you create more then one margin direction use margin 2em 10px 0 0;

Comment (costashush) [reply src/components/Reviews/Review.js]: You rigth I will fix it

**Comment by costashush** [comment]
Ok will fix it

**Comment by costashush** [comment]
fixed the image show component (padding alteration)
added scroll to  reviews

**Pull request merged**